src/components/CheckoutPayment/formOptions.js

```diff
@@ -0,0 +1,50 @@
+// Generates options for mont and year expiry selects
+
+export function months() {
```

Comment:
There is no need to export this function, if it's an internal function to generate months.

Reply:
It's exported for testing purposes.

Reply:
We shouldn't test implementation details ('private' functions), I would just export the 2 main functions and test only those 2 🚀

Reply:
I have to disagree. Those are no private functions in OOP sense. But rather small composable blocks meant for reuse. Can't see any reason not to test them.